lib/instruments.js

```diff
@@ -149,8 +150,8 @@ class Instruments {
     this.proc.stderr.setEncoding('utf8');
     let actOnStderr = (output) => {
       if (this.launchTimeout.afterSimLaunch && output && output.match(/CLTilesManagerClient: initialize/)) {
-        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async () => {
-          await killAllInstruments();
+        this.addSocketConnectTimer(this.launchTimeout.afterSimLaunch, 'afterLaunch', async (done) => {
```

Comment:
Yes, in this case, after socketConnectTimer timeout, we shall kill instruments process (make sure the process was killed) and reject to exit `await launchResultPromise;`
By throw error "ERR_NEVER_CHECKED_IN", so in while loop, we catch this error and retry again

If we just call 
```
if (this.proc) {
  this.proc.kill()
}
```
Some time the instruments process doesn't be killed and we got blocking issue (and leak this instrument process). I think that why in the pass we must call killAllInstruments()

If you don't agree with this solution, I'll try to other way to fix it
Thanks